### PR TITLE
[API Compatibility No.188] Update nll_loss docs for target alias support -part

### PR DESCRIPTION
**Pull request by Aidenwu0209**
## Summary
- Update `nll_loss_cn.rst`: add alias annotation `别名：``target``` for `label`.
- Update `torch.nn.functional.nll_loss.md`: in parameter mapping, clarify Paddle also supports `target`.

## Related PRs
- Paddle: https://github.com/PaddlePaddle/Paddle/pull/78133
- PaConvert: https://github.com/PaddlePaddle/PaConvert/pull/844

## PR Category
User Experience

## PR Types
New features

### CODE REVIEW

**Review by zhwesky2010 — APPROVED** [review]

**Comment by luotao1** [comment]
冲突了
